docs/index.md

```diff
@@ -18,15 +18,120 @@ that include native code, you can take a look at our progress here:
 
 - [Compatibility Status Tracking](tracking.md)
 
-This website also provide documentation and porting guidance - with a focus on
-extension modules using the Python C API, because that's where most of the work
-will be. The following resources should get you started:
+This website also provide documentation and guidance for setting up a
+free-threaded Python development environment and getting code working under the
+free-threaded build.
+
+## Installing and Running Free-threaded Python
+
+The following sections describe how to install free-threaded Python and how
+to run with the GIL disabled and verify that is the case.
 
 - [Installing Free-Threaded Python](installing_cpython.md)
 - [Running Python with the GIL Disabled](running-gil-disabled.md)
-- [Porting Python Packages to Support Free-Threading](porting.md)
-- [Setting up CI](ci.md)
-- [Uncovering Concurrency Issues, Testing and Debugging](debugging.md)
+
+## Updating code to run on free-threaded Python
+
+If you are interested in updating your code to work on free-threaded Python,
+what you need to do depends on your needs and goals. Please select the option
+that best describes you:
+
+??? success "I am writing a Python script or application that uses Python libraries I don't maintain"
+
+    You should start experimenting with free-threaded Python once the libraries
+    you depend on advertise support for free-threading. See [the tracking
+    table](tracking.md) for more details about the status of free-threaded
+    support in Python libraries.
+
+    If your dependencies advertise free-threaded support, good news! You should
```

Comment:
After "good news", I'd insert: "your script or application should run under free-threaded CPython without any issues. If you want to start using the `threading` module to get performance benefits, then"